### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by lex1975**
Added some localization. One bug fixed

### CODE REVIEW

File: src/main/resources/assets/extratrees/lang/ru_RU.lang [hunk 1]
```diff
@@ -58,7 +58,11 @@ extratrees.block.planks.olive=оливы
 extratrees.block.planks.sweetgum=ликвидамбара смолоносного
 extratrees.block.planks.rosewood=розового дерева
 extratrees.block.planks.gingko=гинкго двулопастного
-extratrees.block.planks.pinkivory=умнини
+# extratrees.block.planks.pinkivory=умнини
```
Comment (KorDum) [inline src/main/resources/assets/extratrees/lang/ru_RU.lang]: Why do you commented it?
Localization key is right.

**Review by KorDum — CHANGES_REQUESTED** [review]

**Comment by KorDum** [comment]
Can you add RUS localization for https://github.com/KorDum/Binnie/blob/33f985c2e7c363554cdbd81a1f15d21e66ebea92/src/main/resources/assets/botany/lang/en_US.lang#L16-L123 ?

File: src/main/resources/assets/extratrees/lang/ru_RU.lang (hunk 1 above)
Comment (lex1975) [reply src/main/resources/assets/extratrees/lang/ru_RU.lang]: It is right key (With a space):
extratrees.block.planks.pink ivory=умнини
I believe that this line is not needed:
extratrees.block.planks.pinkivory=умнини
Maybe I'm wrong.

Comment (KorDum) [reply src/main/resources/assets/extratrees/lang/ru_RU.lang]: No. See screenshot. I supplemented and corrected a lot of localization in new version.
image

**Comment by lex1975** [comment]
Can you add RUS localization for https://github.com/KorDum/Binnie/blob/33f985c2e7c363554cdbd81a1f15d21e66ebea92/src/main/resources/assets/botany/lang/en_US.lang#L16-L123 ?

Probably not. I am not aware of the terms of design.

**Comment by KorDum** [comment]
It's a pity. Ok, I'll try to translate myself =)

File: src/main/resources/assets/extratrees/lang/ru_RU.lang (hunk 1 above)
Comment (KorDum) [reply src/main/resources/assets/extratrees/lang/ru_RU.lang]: Please remove this change and I merge PR.

File: src/main/resources/assets/extratrees/lang/ru_RU.lang [hunk 2]
```diff
@@ -58,7 +58,6 @@ extratrees.block.planks.olive=оливы
 extratrees.block.planks.sweetgum=ликвидамбара смолоносного
 extratrees.block.planks.rosewood=розового дерева
 extratrees.block.planks.gingko=гинкго двулопастного
-# extratrees.block.planks.pinkivory=умнини
 extratrees.block.planks.pink ivory=умнини
```
Comment (KorDum) [inline src/main/resources/assets/extratrees/lang/ru_RU.lang]: This is wrong key.
Need without space. See screenshot above.

File: src/main/resources/assets/extratrees/lang/ru_RU.lang [hunk 3]
```diff
@@ -58,7 +58,10 @@ extratrees.block.planks.olive=оливы
 extratrees.block.planks.sweetgum=ликвидамбара смолоносного
 extratrees.block.planks.rosewood=розового дерева
 extratrees.block.planks.gingko=гинкго двулопастного
-extratrees.block.planks.pinkivory=умнини
+extratrees.block.planks.pink ivory=умнини
```
Comment (KorDum) [inline src/main/resources/assets/extratrees/lang/ru_RU.lang]: This is wrong key. Need without space.
See screenshot below.
https://cloud.githubusercontent.com/assets/1331559/26763298/558b19f2-4959-11e7-9ef2-6889bc20705c.png

**Review by KorDum — CHANGES_REQUESTED** [review]

**Pull request merged**